fwutil/lib.py

```diff
@@ -573,11 +577,12 @@ def __validate_platform_schema(self, pcp):
             pcp.chassis_component_map
         )
 
-        self.__validate_component_map(
-            self.SECTION_MODULE,
-            self.module_component_map,
-            pcp.module_component_map
-        )
+        if self.__is_non_smart_switch_modular_chassis:
```

Comment:
We can't disable validation for the smart switch. The validation should be changed for the smart switch to correctly handle components returned by the chassis